src/renderer/coremods/noDevtoolsWarning/index.ts

```diff
@@ -1,32 +1,17 @@
-import Coremod from "../../entities/coremod";
-import { patchPlaintext } from "../../modules/webpack";
+/*
+Plaintext patch version of this coremod:
 
-export default class NoDevtoolsWarningMod extends Coremod<{
-  enabled: boolean;
-}> {
-  public constructor() {
-    super("dev.replugged.coremods.NoDevtoolsWarningMod", "nodevtoolswarning");
-  }
-
-  public start(): void {
-    // nothing
-  }
-
-  public stop(): void {
-    // nop
-  }
+{
+  find: "setDevtoolsCallbacks",
+  replacements: [
+    {
+      match: /if\(.{0,10}\|\|"0.0.0"!==.{0,2}\.remoteApp\.getVersion\(\)\)/,
+      replace: "if(false)"
+    }
+  ]
+}
+*/
```

Comment:
It's not good practice to comment out code like this. That's what Git history is for.
```suggestion
```

Reply:
Fixed